frontend/App.tsx

```diff
@@ -1,62 +1,74 @@
+} from '@react-navigation/native'
+import { createNativeStackNavigator } from '@react-navigation/native-stack'
+import * as NavigationBar from 'expo-navigation-bar'
 
-import merge from 'deepmerge';
+import merge from 'deepmerge'
 
-import {LoginContext} from './Context';
-import {useCookies} from "react-cookie";
+import { LoginContext } from './Context'
+import { useCookies } from 'react-cookie'
 
-import LoginScreen from "./Screens/Login";
-import OnboardingScreen from "./Screens/Onboarding";
-import HomeTabs from './HomeTabs';
-import { StatusBar } from 'react-native';
+import LoginScreen from './screens/Login'
+import OnboardingScreen from './screens/Onboarding'
+import HomeTabs from './HomeTabs'
+import { StatusBar } from 'react-native'
 
-const Stack = createNativeStackNavigator();
+const Stack = createNativeStackNavigator()
 
 const App = (): JSX.Element => {
-    const [cookies, removeCookie] = useCookies(['loginCookie']);
-    const [isSignedIn, setIsSignedIn] = React.useState(cookies.loginCookie !== undefined)
-
-    const appContextValue = useMemo(
-        () => ({
-            isSignedIn,
-            setIsSignedIn,
-        }),
-        [isSignedIn]
-    );
-    const CombinedDarkTheme = merge(PaperDarkTheme, NavigationDarkTheme);
-
-    const theme = {
-        ...CombinedDarkTheme,
-        "colors": {
-            ...CombinedDarkTheme.colors,
-            "background": "#222023",
-            "accent": "#BDBCBD",
-            "primary": "#008080"
-        }
-    }
-    NavigationBar.setBackgroundColorAsync("black");
-
-    return (
-        <PaperProvider theme={theme}>
-            <StatusBar animated={true} hidden={false} />
-            <LoginContext.Provider value={appContextValue}>
-                <NavigationContainer theme={theme}>
-                    <Stack.Navigator id="root-stack-navigator"
-                        screenOptions={{
-                            headerShown: false
-                        }}>
-                            <Stack.Screen name="login" component={LoginScreen}/>
-                            <Stack.Screen name="onboarding" component={OnboardingScreen}/>
-                            <Stack.Screen name="home-tab-navigation" component={HomeTabs}/>
-                    </Stack.Navigator>
-                </NavigationContainer>
-            </LoginContext.Provider>
-        </PaperProvider>
-    )
+  const [cookies, removeCookie] = useCookies(['loginCookie'])
+  const [isSignedIn, setIsSignedIn] = React.useState(
+    cookies.loginCookie !== undefined,
+  )
+
+  const appContextValue = useMemo(
+    () => ({
+      isSignedIn,
+      setIsSignedIn,
+    }),
+    [isSignedIn],
+  )
+  const CombinedDarkTheme = merge(PaperDarkTheme, NavigationDarkTheme)
+
+  const theme = {
+    ...CombinedDarkTheme,
+    colors: {
+      ...CombinedDarkTheme.colors,
+      background: '#222023',
+      accent: '#BDBCBD',
+      primary: '#008080',
+    },
+  }
+  NavigationBar.setBackgroundColorAsync('black')
```

Comment:
Need to check if the platform is android since expo throws a warning on ios devices.